.ci/bump-version.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+AGENT_VERSION="${1?Missing the APM python agent version}"
+
+sed -ibck "s#elastic-apm==.*#elastic-apm==${AGENT_VERSION}#g" requirements.txt
+
+# Commit changes
+git add requirements.txt
+git commit -m "Bump version ${AGENT_VERSION}"
```

Comment:
doesn't this have to be pushed to elastic/opbeans-python? Or does that happen elsewhere?

Reply:
Ah, I guess "elsewhere" is here

https://github.com/elastic/apm-agent-python/pull/656/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R272

Reply:
Good point, I decoupled the commit from the push. The main reason is about the responsibility, the one in charge for the bump should be caring about committing the new files. Therefore the pipeline orchestrator in the `apm-agent-python` should care for calling this script and push, see https://github.com/elastic/apm-agent-python/pull/656/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R270-R274